app.py

```diff
@@ -1023,14 +1050,36 @@ def handle_option(self, option):
                     print("Invalid choice. Try again.")
                     update = False
 
-        # Exit
+        # Load Product File
         elif option == '3':

```

Comment:
Move the `Load Product` option to the Settings menu:
https://docs.google.com/document/d/1T4359w0cE_PlkcQXzoLDOFQLIQcNHoX14k17_wYWQ9Y/edit#heading=h.51mtkh23489h